test/integration/globals/error.js

```diff
@@ -1,7 +1,7 @@
 context('Global Error Page', function() {
   specify('Displaying and navigating the global error page', function() {
     cy
-      .visit('/');
+      .visit('/nothinghere');
```

Comment:
actually what's happening here is that the `navigate` trigger is happening prior to the router starting because we introduced an async load before starting the router (a pattern we didn't have in mainapp because the bootstrap came with the initial html payload)   _but_ it looks like the history service still handles the back button correctly even if there's nothing to go back to.. _so_ lines 6-11 should be removed and line 4 should be `/404`

Perhaps the first test should be
```
cy
      .url()
      .should('contain', '404');
```